app/src/main/res/layout/search_selector.xml

```diff
@@ -14,7 +14,7 @@
         android:layout_height="28dp"
         android:layout_marginTop="14dp"
         android:layout_marginHorizontal="8dp"
-        app:cardBackgroundColor="?attr/layer2"
+        app:cardBackgroundColor="#ffffff"
```

Comment:
similar to above, just want to make sure this is correct for both light/dark

Reply:
Don't think this is intended change either since it goes against the designs.

Reply:
This is intended.  Since we're on this topic, @gabrielluong is there a better white and black colour I should be using here rather than hardcoding?  I need color that stays white / black even with theme changes.